cartographer/mapping_2d/local_trajectory_builder.cc

```diff
@@ -228,15 +228,18 @@ LocalTrajectoryBuilder::AddHorizontalRangeData(
     return nullptr;
   }
 
+  // indices of submaps_ where new scans are inserted are decided here...
```

Comment:
Style: Use full sentences, e.g. combine the three comments to one comment here:
```
  // Querying matching and insertion indices must be done here before calling
  // InsertRangeData() since the queried values are valid for next insertion.
```